### PR TITLE
#2724 added message after commit

### PR DESCRIPTION
**Pull request by Hir0ki**
* A popup_to_inform is used to show to user that the commit is successful.

ABAPGIT_commit_successful

### CODE REVIEW

**Comment by flaiker** [comment]
Seems a bit intrusive to me. A simple `MESSAGE ... TYPE 'S'` might be better? Not sure.

**Comment by larshp** [comment]
Yea, would like not to do extra button clicks

**Comment by Hir0ki** [comment]
I changed the popup to a message 
ABAPGIT_COMMIT_MSG

**Comment by FreHu** [comment]
Thanks for contributing! This will resolve #2724.

File: src/ui/zcl_abapgit_gui_page_commit.clas.abap [hunk 1]
```diff
@@ -329,11 +329,13 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_COMMIT IMPLEMENTATION.
 
   METHOD zif_abapgit_gui_event_handler~on_event.
 
-    DATA: ls_commit TYPE zcl_abapgit_services_git=>ty_commit_fields.
+    DATA: ls_commit TYPE zcl_abapgit_services_git=>ty_commit_fields,
+          li_popups TYPE REF TO zif_abapgit_popups.
```
Comment (FreHu) [inline src/ui/zcl_abapgit_gui_page_commit.clas.abap]: This variable is unused now.

**Comment by Hir0ki** [comment]
> 
> 
> see comment by FreHu above

fixed the issue.

**Comment by larshp** [comment]
test

**Comment by larshp** [comment]
@Hir0ki looks like you have deleted the source branch? Hence I cannot merge the changes
image

**Comment by Hir0ki** [comment]
Oh sry, I didn't knew that you would be able to merge the change.  
I will create a new pull request.

**Pull request closed without merging**